### PR TITLE
Fixes #36648 - Show desc and publish date on component CV add/edit modal

### PR DESCRIPTION
**Pull request by sjha4**
#### What are the changes introduced in this pull request?
Add 'published_at' in version label when selecting component CV version to add to composite CV.
#### What are the testing steps for this pull request?
Create some content views and publish. Add descriptions to some/all when publishing. You can also edit desc on published versions.
Create a composite content view and play around with adding/editing/bulk adding components to the composite CV.
Make sure all three(single add/edit and bulk add) look like sketch here: 
Screenshot from 2023-08-07 15-03-31

### CODE REVIEW

**Review by jeremylenz — COMMENTED** [review]
(For posterity): The dropdown seems to be scrolling the modal, but instead should extend outside of it. I think PF has a way you can make this happen.

File: webpack/scenes/ContentViews/Details/ComponentContentViews/ComponentContentViewAddModal.js [hunk 1]
```diff
@@ -37,14 +37,17 @@ const ComponentContentViewAddModal = ({
...
   const versionsLoading = componentStatus === STATUS.PENDING;
 
   useEffect(() => {
     if (!versionsLoading && componentDetails) {
       const { name, versions } = componentDetails;
       const versionMutable = versions;
       setCvName(name);
-      const opt = versionMutable.map(item => ({ value: item.id, label: __(`Version ${item.version}`) }));
+      const opt = versionMutable.map(item => ({
+        value: item.id, label: __(`Version ${item.version}`), description: item.description, publishedAtWords: __(` (${item.published_at_words} ago)`),
```
[the remark this answers is not in the document]
Comment (sjha4) [reply webpack/scenes/ContentViews/Details/ComponentContentViews/ComponentContentViewAddModal.js:49]: I did leave that space there intentionally to have spacing between Version label and the published date.

**Comment by sjha4** [comment]
Updated to make scroll appear on top of modal. The prop was `menuAppendTo="parent"`

File: webpack/scenes/ContentViews/Details/ComponentContentViews/ComponentContentViewAddModal.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (jeremylenz) [reply webpack/scenes/ContentViews/Details/ComponentContentViews/ComponentContentViewAddModal.js:49]: The space should be before the parenthesis, not after.  Looks weird with a space after an opening parenthesis.

[the remark this answers is not in the document]
Comment (sjha4) [reply webpack/scenes/ContentViews/Details/ComponentContentViews/ComponentContentViewAddModal.js:49]: So this:  
```
__(`Version ${item.version}` )
```

[the remark this answers is not in the document]
Comment (jeremylenz) [reply webpack/scenes/ContentViews/Details/ComponentContentViews/ComponentContentViewAddModal.js:49]: Just want you to fix this:
image

[the remark this answers is not in the document]
Comment (sjha4) [reply webpack/scenes/ContentViews/Details/ComponentContentViews/ComponentContentViewAddModal.js:49]: Oh wait..The first ( there is  not part of the displayed screen.. The displayed string is just this: 
` (${item.published_at_words} ago)`

**Review by jeremylenz — COMMENTED** [review]
Dropdowns are looking beautiful now and extending past the modals properly!

One more issue: I see this when I check the checkbox:

image

**Comment by sjha4** [comment]
Pushed fixes to tests 🍏 🤞🏼

**Review by jeremylenz — APPROVED** [review]
Looking great @sjha4, thanks!

ACK :+1:

**Pull request merged**